python/graphstorm/eval/evaluator.py

```diff
@@ -108,9 +108,386 @@ def get_val_score_rank(val_score, val_perf_rank_list, comparator):
     return rank
 
 
+class GSgnnBaseEvaluator():
```

Comment:
This class is very close to GSgnnInstanceEvaluator, why not just rename GSgnnInstanceEvaluator and update it?

Reply:
I want to make sure the existing code will not be broke, and then replace the deprecated Evaluators one by one.